decompose/src/webMain/kotlin/com/arkivanov/decompose/Utils.kt

```
@@ -1,8 +1,21 @@
 internal val Json =
     Json {
         allowStructuredMapKeys = true
     }
+
+internal fun SerializableContainer.encodeToJson(): JsonString =
+    JsonString(Json.encodeToString(SerializableContainer.serializer(), this))
+
+internal fun JsonString.decodeContainer(): SerializableContainer? =
+    try {
+        Json.decodeFromString(SerializableContainer.serializer(), value)
+    } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)